oidc_example/simple_rp/requirements.txt

```diff
@@ -1,4 +1,3 @@
-oic==0.7.6
 pyjwkest==1.0.1
```

Comment:
This should be included in the installation and is probably extra.

Reply:
Should this be worded "should not be included"?

Reply:
Yes, sorry. Applies to both comments.